mithril.js

```diff
@@ -1081,8 +1081,14 @@
 			//
 			// #348 don't set the value if not needed - otherwise, cursor
 			// placement breaks in Chrome
+			// #1252 likewise when `contenteditable` is set on an element.
 			try {
-				if (tag !== "input" || node[attrName] !== dataAttr) {
+				if (
+					tag !== "input" && !node.isContentEditable
+					/* eslint-disable eqeqeq */
+					|| node[attrName] != dataAttr
```

Comment:
@pygy 

I had changed this myself a while back (0.2.0-0.2.2 IIRC), but after I got ESLint running on this file. I didn't realize that the loose equality was necessary at the time (apparently it is, because Chrome is a fail).

I'll take the blame on this one.